src/react/ui-elements/Input.ts

```diff
@@ -42,15 +42,20 @@ const Input = styled.input`
     width: -webkit-fill-available; /* Mozilla-based browsers will ignore this. */
     width: fill-available;  
   `}
-    :hover:enabled {
+  :hover:enabled {
     border: ${spacing.sp1} solid ${(props) => props.theme.brand?.border};
   }
 
   :focus {
     outline: none;
     border: ${spacing.sp1} solid ${(props) => props.theme.brand?.secondary};
   }
-
+  ${(props) =>
+    props.disabled &&
+    `  pointer-events: auto;
+cursor: not-allowed;
+opacity: 0.5;
+`}
```

Comment:
Ideally this should be in core ui. Also @Cuervino I'm wondering if similarly as select component we should add the disabled icon at the extreme right of the input

Reply:
I will create a debt ticket to migrate the input component with the one we have in core-ui.